compiler/rustc_codegen_llvm/src/asm.rs

```diff
@@ -244,6 +244,7 @@ impl<'ll, 'tcx> AsmBuilderMethods<'tcx> for Builder<'_, 'll, 'tcx> {
                 InlineAsmArch::Msp430 => {
                     constraints.push("~{sr}".to_string());
                 }
+                InlineAsmArch::M68k => {}
```

Comment:
This should clobber `ccr`. Also document this in the unstable book under the `preserves_flags` section.